nnpdf_data/nnpdf_data/theory.py

```diff
@@ -0,0 +1,51 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class TheoryCard:
+    ID: int # ID number of the theory
+    PTO: int # Perturbative order (0 = LO, 1 = NLO, 2 = NNLO ...)
+    FNS: str # Flavor number scheme (i.e. FONLL-C)
+    DAMP: int # Whether a damping function is applied or not for FONLL
+    DAMPPOWERc: int = None # Power of the damping factor in FONLL for the c
+    DAMPPOWERb: int = None # Power of the damping factor in FONLL for the b
+    IC: int # 0 = perturbative charm only , 1 = intrinsic charm allowed
+    ModEv: str # DGLAP evolution solution method (EXA or TRN)
+    IterEv: int = None # Number of iterations for the evolution of the PDF. Defaults to 40 when ModEv = EXA
+    ModSV: str = None # Scale variations method in EKO (expanded or exponentiated)
+    XIR: float # Renormalization scale over the hard scattering scale ratio
+    XIF: float # Factorization scale over the hard scattering scale ratio
+    NfFF: int # Number of active flavors, only for FFNS or FFN0 schemes
+    nfref: int # Number of active flavors at Qref
+    nf0: 3 # Number of active flavors at the parametrization scale Q0
```

Comment:
```suggestion
    nf0: int = 3 # Number of active flavors at the parametrization scale Q0
```
(and put at the end)